src/pages/catalog-page/catalog-options/components/select/handle-field-options.ts

```diff
@@ -0,0 +1,9 @@
+import { catalogStore } from '../../../../../store/catalog-store';
+
+export const handleFieldOptions = (
+  event: React.ChangeEvent<HTMLSelectElement>
+) => {
+  const newField = event.target.value;
+  const newOrder = newField === 'price' ? 'asc' : 'asc';
```

Comment:
```suggestion
  const newOrder = newField === 'price' ? 'asc' : 'asc';
```
наверное ты не так хотела. вообще у тебя почти такая же функция в теле компонента. тащи наверное и эту туда же. как раз и ордер сможешь сделать как во второй